mypy.ini

```diff
@@ -7,7 +7,7 @@ disallow_untyped_defs = True
 warn_unused_ignores = False
 disallow_any_generics = True
 
-exclude = pyvenv|(_test\.py|test_.*\.py)|_pb2\.py|_pb2\.pyi
+exclude = pyvenv|(_test\.py|test_.*\.py)|_pb2\.py|_pb2\.pyi|admin_app/
```

Comment:
We should have mypy for streamlit apps I think

Reply:
so I moved our streamlit code from cortex repo which's under 3.10 and here we use python 3.8 which i need to update a lot of places for the lint to work... if you don't mind i'll push lint to later. And i think for larger changes, make sense to do it in cortex repo first and then move to here.